session/src/main/scala/akka/stream/alpakka/cassandra/scaladsl/CassandraSession.scala

```diff
@@ -301,68 +275,13 @@ final class CassandraSession(
    * if any.
    */
   def selectOne(stmt: String, bindValues: AnyRef*): Future[Option[Row]] = {
-    val bound: Future[BoundStatement] = prepare(stmt).map(
-      ps =>
-        if (bindValues.isEmpty) ps.bind()
-        else ps.bind(bindValues: _*))
-    bound.flatMap(bs => selectOne(bs))
+    bind(stmt, bindValues).flatMap(bs => selectOne(bs))
   }
 
-  private class SelectSource(stmt: Future[Statement[_]]) extends GraphStage[SourceShape[Row]] {
```

Comment:
We should do some performance benchmark of this change. Great to get rid of this, but we should know that it doesn't reduce performance. For example the Publisher will add an async boundary for each emitted row while this custom GraphStage will be fused with the rest of the stream.